entity-framework/core/what-is-new/ef-core-7.0/breaking-changes.md

```diff
@@ -286,6 +287,39 @@ modelBuilder
     .OnDelete(DeleteBehavior.ClientCascade);
 ```
 
+<a name="sqliteretries"></a>
+
+### Higher chance of busy/locked errors on SQLite when not using write-ahead logging
+
+#### Old behavior
+
+Previous versions of the SQLite provider saved changes via a less efficient technique which was able to automatically retry when the table was locked/busy and write-ahead logging (WAL) was not enabled.
+
+#### New behavior
+
+By default, EF Core now saves changes via a more efficient technique, using the RETURNING clause. Unfortunately, this technique is not able to automatically retry when busy/locked. In a multi-threaded application (like a web app) not using write-ahead logging, it is common to encounter these errors.
+
+#### Why
+
+The simplifications and performance improvements linked to the new method are significant enough that it's important to bring them to users by default. Databases created by EF Core also enable write-ahead logging by default. The SQLite team also recommends enabling write-ahead logging by default.
+
+#### Mitigations
+
+If possible, you should enable write-ahead logging on your database by executing the following command.
```

Comment:
I'd add a note reiterating that if you use EF to create the database (presumably the very common case), nothing needs to be done and you're fine.